app/assets/stylesheets/components/_inputs.scss

```diff
@@ -6,8 +6,8 @@
   @include icomoon;
   content: $icon-chevron-thin-down;
   font-size: 2.5rem;
-  bottom: 0.25em;
-  right: 0.5em;
+  bottom: $em-spacer-3;
+  right: $em-spacer-5;
```

Comment:
What's the story around when to use the sass variables vs css variables?

Reply:
I went with whatever was documented: https://primer.style/css/support/variables
It seems like all the colors are currently CSS custom properties at the moment.